roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -64,23 +71,15 @@
   become: true
   template:
     src: "{{ role_path }}/templates/ceph-osd.service.j2"
-    dest: /var/lib/ceph/ceph-osd@.service
+    dest: /etc/systemd/system/ceph-osd@.service
```

Comment:
It will worj=k that is why this unit file is templated with ceph-osd@.service. When you do systemctl enable then systemd automatically makes those symlinks.